src/instance.rs

```diff
@@ -1,13 +1,13 @@
 use crate::conversion::PyTryFrom;
 use crate::err::{self, PyDowncastError, PyErr, PyResult};
-use crate::gil;
 use crate::pycell::{PyBorrowError, PyBorrowMutError, PyCell};
 use crate::pyclass::boolean_struct::{False, True};
 use crate::types::{PyDict, PyString, PyTuple};
 use crate::{
-    ffi, AsPyPointer, FromPyObject, IntoPy, IntoPyPointer, PyAny, PyClass, PyClassInitializer,
-    PyRef, PyRefMut, PyTypeInfo, Python, ToPyObject,
+    ffi, FromPyObject, IntoPy, IntoPyPointer, PyAny, PyClass, PyClassInitializer, PyRef, PyRefMut,
+    PyTypeInfo, Python, ToPyObject,
 };
+use crate::{gil, AsPyPointer};
```

Comment:
This looks unintentional?

Reply:
Yes, I removed `AsPyPointer` from line 7 but then found there are several implementations in this file which use it, so left it here for now. I'll revert this diff to just leave `AsPyPointer` imported on line 7.